tensorflow/lite/delegates/gpu/common/model_builder.cc

```diff
@@ -1406,27 +1409,36 @@ class MulOperationParser : public TFLiteOperationParser {
         input_tensor0 = 1;
         input_tensor1 = 0;
       }
-      return ParseApplyMask(input_tensor0, input_tensor1, graph, reader);
+      RETURN_IF_ERROR(ParseApplyMask(input_tensor0, input_tensor1, node, graph, reader));
+    }
+    else
+    {
```

Comment:
Please, follow the google code style:
} else {

Reply:
Thanks

Reply:
I'm still seeing

}
else
{